src/algebra/big_operators/basic.lean

```diff
@@ -1649,8 +1649,78 @@ end list
 
 namespace multiset
 
+lemma disjoint_list_sum_left {a : multiset α} {l : list (multiset α)} :
+  (∀ b ∈ l, multiset.disjoint b a) ↔ multiset.disjoint l.sum a :=
+begin
+  induction l with b bs ih,
+  { simp only [zero_disjoint, list.not_mem_nil, is_empty.forall_iff, forall_const, list.sum_nil], },
+  { simp_rw [list.sum_cons, disjoint_add_left, list.mem_cons_iff, forall_eq_or_imp],
+    simp [and.congr_left_iff, iff_self, ih], },
+end
+
+lemma disjoint_list_sum_right {a : multiset α} {l : list (multiset α)} :
+  (∀ b ∈ l, multiset.disjoint a b) ↔ multiset.disjoint a l.sum :=
+begin
+  simp_rw @disjoint_comm _ a _,
+  exact disjoint_list_sum_left,
+end
+
+lemma disjoint_sum_left {a : multiset α} {i : multiset (multiset α)} :
+  (∀ b ∈ i, multiset.disjoint b a) ↔ multiset.disjoint i.sum a :=
+quotient.induction_on i $ λ l, begin
+  rw [quot_mk_to_coe, multiset.coe_sum],
+  exact disjoint_list_sum_left,
+end
+
+lemma disjoint_sum_right {a : multiset α} {i : multiset (multiset α)} :
+  (∀ b ∈ i, multiset.disjoint a b) ↔ multiset.disjoint a i.sum :=
+begin
```

Comment:
Does `simpa only [disjoint_comm] using disjoint_sum_left` work here?

Reply:
It actually works in all the ```right``` proofs. I guess I still have to learn the power of simpa!